src/common/mfu_io.c

```diff
@@ -1489,6 +1493,7 @@ struct dirent* mfu_file_readdir(DIR* dirp, mfu_file_t* mfu_file)
     }
 }
 
+#ifdef DCOPY_USE_XATTRS
```

Comment:
Why use `#ifdef` here but `#if` above?

Reply:
Mistype... This should be #if DCOPY_USE_XATTRS like everywhere else. Thanks :)